web-common/src/features/models/workspace/ModelWorkspaceHeader.svelte

```diff
@@ -45,12 +41,19 @@
 
   let contextMenuOpen = false;
 
-  const onExport = async (exportExtension: "csv" | "parquet") => {
-    // TODO: how do we handle errors ?
-    window.open(
-      `${RuntimeUrl}/v1/instances/${$runtimeStore.instanceId}/table/${modelName}/export/${exportExtension}`
-    );
-  };
+  $: availableDashboards = useRuntimeServiceListCatalogEntries(
+    $runtimeStore.instanceId,
+    { type: "OBJECT_TYPE_METRICS_VIEW" },
+    {
+      query: {
+        select(data) {
+          return data?.entries?.filter(
+            (entry) => entry?.metricsView?.model === modelName
+          );
+        },
+      },
+    }
+  );
```

Comment:
We can move this to `svelte-query/dashbords.ts` (which will be later moved to `features/dashboards/selectors.ts`)